step6_train_chrombpnet_model.sh

```diff
@@ -14,7 +14,13 @@ fold=$5
 bias_model=$6
 output_dir=$7
 data_type=$8
-logfile=$9
+seed=$9
```

Comment:
suggestion for the default seed: 

```suggestion
seed=${9-1234}
```